gnovm/stdlibs/std/banker.go

```diff
@@ -30,6 +31,9 @@ const (
 	btRealmSend
 	// Can issue and remove realm coins.
 	btRealmIssue
+
+	// regexp for denom format
+	denomRegex = "[a-z][a-z0-9]{2,15}"
```

Comment:
And also pleease move it out of this const block

Const blocks are meant for "similar" values; the `denomRegex` has nothing to do with the banker type.

Reply:
5311666f3bd5b0b951e805a9390773b4719b28e8